core/server/src/main/java/alluxio/master/AbstractMaster.java

```diff
@@ -61,6 +62,8 @@
   /** The journal writer for when the master is the leader. */
   private JournalWriter mJournalWriter = null;
 
+  private AsyncJournalWriter mAsyncJournalWriter = null;
```

Comment:
javadoced, but this cannot be final, since it can be changed when it becomes a master.